app/views/registration_wizard/shared/_question_page.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :title do %>
   <%= form.errors.present? ? "Error: " : nil %>
-  <%= t("helpers.#{form.question.title_locale_type}.registration_wizard.#{form.question.name}") %>
+  <%= t("helpers.#{form.question.title_locale_type}.registration_wizard.#{form.question.name_locale_key}") %>
```

Comment:
Locales within questions are an interesting problem. I definitely agree those should follow form names rather than field name, but would be great to just make them available via `t('.key')`. I'd probably move them out of `helpers` into their own `questions` scope?

Reply:
I have a second PR coming up to change FindChildcareProvider that moves all these question/hint locale strings into a method on the question. That way we can do this sort of refactor to move the strings somewhere else.